Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,81 @@
+var humanChoice = "";
+var humanScore = 0;
+var computerScore = 0;
+
+
+$('.choice').click(function() {
+   humanChoice = this.id;
+
+	 var choices = ["rock", "paper", "scissors"];
+
+	 var computerChoice = choices[Math.floor(Math.random() * choices.length)];
+
+   compare(humanChoice, computerChoice);
+
+	 document.querySelector("#computerScore").innerHTML = computerScore;
+	 document.querySelector("#humanScore").innerHTML = humanScore;
+
+
+});
+
+
+
+function compare (humanChoice, computerChoice){
+  if (humanChoice === computerChoice) {
+
+    document.querySelector("#status").innerHTML = "It's a tie!";
+		}
+
+  	else if (humanChoice === "rock") {
+
+
+ 		if (computerChoice === "scissors") {
+
+    humanWins();
+
+		} else {
+
+		botWins();
+		}
+	}
+
+
+
+		else if (humanChoice === "paper") {
```

Comment:
Try to keep your indentation and spacing tight as it makes it clearer which pieces of logic are working together. For example here it took me a while to work out that this is the `else if` that goes with the `if` on line 24